neuralpredictors/layers/cores/conv2d.py

```diff
@@ -332,7 +336,7 @@ def set_batchnorm_type(self):
 
     def add_first_layer(self):
         layer = OrderedDict()
-        layer["conv"] = HermiteConv2D(
+        layer["hermite_conv"] = HermiteConv2D(
```

Comment:
same with the HermiteConv. Previously it was also just named `conv`.

https://github.com/sinzlab/neuralpredictors/blob/862198b1790c3a1779a87930dbbac8ef05370e0c/neuralpredictors/layers/cores.py#L360